src/hooks/useBrevets.ts

```diff
@@ -0,0 +1,63 @@
+import { useMemo } from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+
+export type Chapter = 'Toronto' | 'Huron' | 'Ottawa' | 'Simcoe'
+
+export type RideType = 'brevet' | 'permanent' | 'fleche' | 'populaire'
+
+export type Brevet = {
+  chapter: Chapter
+  distance: number
+  event: RideType
+  id: string
+  organizer: string
+  route: string
+  rwgpsUrl: string
+  rwgpsId: number
+  season: number
+  startLocation: string
+  date: Date
+}
+
+const today = new Date(Date.now())
+
+const brevetQuery = graphql`
```

Comment:
Update to apollo https://github.com/jlengstorf/gatsby-with-apollo/blob/master/src/pages/index.js